examples/applications/plot_model_complexity_influence.py

```diff
@@ -16,17 +16,21 @@
 
 We will model the complexity influence on three different estimators:
     - :class:`~sklearn.linear_model.SGDClassifier` (for classification data)
-      which implements stochastic gradient descent learning;
+      which implements stochastic gradient descent learning.
+      See example for comparison with different penalties:
+      :ref:`sphx_glr_auto_examples_linear_model_plot_sgd_penalties.py`
```

Comment:
I think this example would be better placed in the doc-strings for [SGDClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html#sklearn.linear_model.SGDClassifier) and [SGDRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html#sklearn.linear_model.SGDRegressor).

Reply:
Hi @virchan, thanks for the suggestion! I checked, and the reference to the example is already in the appropriate spot within the doc-strings for `SGDClassifier` and `SGDRegressor`. This doesn't require any further changes.